src/app/shared/components/template/services/template-variables.service.ts

```diff
@@ -175,6 +175,17 @@ export class TemplateVariablesService extends AsyncServiceBase {
     context: FlowTypes.TemplateRowEvalContext
   ) {
     const fullExpression = evaluators[0].fullExpression;
+
+    // HACK: Ensure expression is translated (evaluators may be created from untranslated strings in some cases)
```

Comment:
Not sure this is actually a hack. Can probably drop from the comment prefix assuming we're able to add a test case that validates the use